packages/test/stochastic-test-utils/src/test/generators.spec.ts

```diff
@@ -43,26 +42,6 @@ async function assertAsyncGeneratorProduces<T>(generator: AsyncGenerator<T, void
     assert.deepEqual(actual, results);
 }
 
-class Counter<T> {
-    private readonly counts = new Map<T, number>();
-
-    public increment(t: T): void {
-        this.counts.set(t, this.get(t) + 1);
-    }
-
-    public get(t: T): number {
-        return this.counts.get(t) ?? 0;
-    }
-
-    public entries(): Iterable<[T, number]> {
-        return this.counts.entries();
-    }
-
-    public values(): Iterable<number> {
-        return this.counts.values();
-    }
-}
-
```

Comment:
I moved Counter<T>() and computeChiSquared to 'utils.ts' so I could use them as well.